test/compflags/checks/checks.good

```diff
@@ -0,0 +1 @@
+truetruetruetrue
```

Comment:
This file is also unused and outdated, but I'd suggest renaming checks-on.good to checks.good and using it for "the checks on" cases since it reflects the compiler's default (i.e., someone just browsing the test and looking for its .good without looking at the compopts file would see what they should expect).